### ISSUE TITLE
Font issue

### ISSUE THREAD

**Issue by lineldcosta** [tonchik-tm/pdf-to-html#6]
Looks like it doesnt set the fonts by default it is setting font as times? is it library issue?

**Comment by selva12** [comment]
Yes, when i am convert PDF to HTML i am also having font issue. Font family is changed. How we can fix that?

**Comment by jamesdordoy** [comment]
Hello,

Just wondered if anyone had got any further with this as it looks like poppler-utils doesn't work very well with custom fonts.
